packages/assets/scripts/createSynMigrationIconLibrary.js

```diff
@@ -0,0 +1,322 @@
+  person_remove_alt_1: 'person_remove',
+  personal_video: 'monitor',
+  phone: 'call',
+  phonelink: 'devices',
+  photo_size_select_actual: 'wallpaper',
+  pix: 'MISSING', // @todo: No idea what this was
+  place: 'location_on',
+  play_circle_filled: 'play_circle',
+  plus_one: 'exposure_plus_1',
+  poll: 'ballot',
+  portrait: 'account_box',
+
+  query_builder: 'alarm', // Set to alarm because its the same as access_alarm
+  question_answer: 'forum',
+  queue: 'add_to_queue',
+
+  remove_circle: 'do_not_disturb_on_total_silence', // @todo: This still does not quite fit
+  remove_circle_outline: 'do_not_disturb_on_total_silence', // @todo: This still does not quite fit
+  remove_red_eye: 'visibility',
+  replay_circle_filled: 'replay',
+  report_gmailerrorred: 'dangerous',
+  report_problem: 'warning',
+  restore: 'history',
+  room: 'location_on',
+
+  save_alt: 'file_download',
+  sd_storage: 'sd_card',
+  security_update: 'install_mobile',
+  sentiment_satisfied_alt: 'sentiment_satisfied',
+  settings_input_composite: 'settings_input_component',
+  shop_2: 'shop_two',
+  shortcut: 'turn_right',
+  signal_cellular_no_sim: 'sd_card_alert',
+  signal_wifi_4_bar_lock: 'MISSING', // @todo: Maybe use signal_wifi_4_bar for this only?
+  signal_wifi_connected_no_internet_4: 'signal_wifi_bad',
+  signal_wifi_statusbar_4_bar: 'signal_wifi_4_bar',
+  signal_wifi_statusbar_connected_no_internet_4: 'signal_wifi_statusbar_not_connected', // No connected now has question mark instead of exclamation
+  sim_card_alert: 'sd_card_alert',
+  source: 'topic', // Same icon, different name
+  star_border: 'grade',
+  star_border_purple500: 'grade',
+  star_purple500: 'grade',
+  store_mall_directory: 'mall',
+  system_security_update: 'install_mobile',
+  system_security_update_good: 'MISSING',
+  system_security_update_warning: 'perm_device_information',
+
+  tag_faces: 'sentiment_satisfied', // @todo: There are also others, search "smil"
+  terrain: 'landscape',
+  textsms: 'sms',
+  thumb_down_alt: 'thumb_down',
+  thumb_down_off_alt: 'thumb_down_off',
+  thumb_up_alt: 'thumb_up',
+  thumb_up_off_alt: 'thumb_up_off',
+  time_to_leave: 'directions_car', // @todo: Same icon, different name
+  try: 'reviews',
+  tungsten: 'lightbulb',
+  turned_in: 'bookmark',
+  turned_in_not: 'bookmark_remove',
+
+  warning_amber: 'warning',
+  watch_later: 'schedule',
+  wb_cloudy: 'cloud_queue',
+  work_off: 'enterprise_off',
+};
+
+const STATUS = {
+  MAPPED_BY_REVERSING: 'MAPPED_BY_REVERSING',
+  MAPPED_DIRECTLY: 'MAPPED_DIRECTLY',
+  MAPPED_STATICALLY: 'MAPPED_STATICALLY',
+  MAPPED_WITHOUT_SUFFIX: 'MAPPED_WITHOUT_SUFFIX',
+  NOT_FOUND_IN_OLD: 'NOT_FOUND_IN_OLD',
+  UNKNOWN: 'UNKNOWN',
+};
+
+/**
+ * @typedef {Object} IconMappingResult
+ * @property {string} name The old icon name
+ * @property {string} status The mapping status
+ * @property {string | null} newName The new icon name, or null if not found
+ */
+
+const keys2015 = Object.keys(i2015);
```

Comment:
should be 2018 and not 2015


```suggestion
const keys2018 = Object.keys(i2018);
```